app/components/settings/billing-page/membership-section.hbs

```diff
@@ -0,0 +1,68 @@
+<Settings::FormSection @title="Membership" @description="Your CodeCrafters membership details" data-test-membership-section>
+  {{#if @user.hasActiveSubscription}}
+    <Settings::BillingPage::StatusPill @color="green">
```

Comment:
Added note below, but let's move the `mb-4` up here so that the spacing between elements in this template is managed here instead of inside the child component